app/controllers/home_controller.rb

```diff
@@ -2,5 +2,6 @@ class HomeController < ApplicationController
   after_action :verify_authorized, except: [:index]
 
   def index
+    redirect_to current_user.after_sign_in_path if user_signed_in?
```

Comment:
I don't necessarily agree, since then the root page is blank if you visit it and you're logged in.